userapp/.env

```diff
@@ -0,0 +1 @@
+PORT=3003
```

Comment:
is there a reason why this isn't ignored?

Reply:
I did not add .env into gitignore. Added!